Add release 6.2 post (#226)
PrismLauncher/prismlauncher.org: src/news/release-6.2.md

---
title: Prism Launcher Release 6.2, now available
description: 90% bug free!
date: 2023-02-03
release_version: "6.2"
minimum_macos_version: 10.15.0
mac_signature: 76S99OXja/WJbqYQvV9BoEAlTV4khWeZWCjcXKFHnXkgkZD18yXcBXAdY/BK0dhQrX1IbHZu3JKhEVyjFwKmCA==
tags:
  - release
---

Hello!

This is a small update that fixes some issues, most notably a nasty issue when importing .mrpack file.
Thanks to all contributors!

## Changelog

### Added

- Make use of logging categories for authentication data logging by [@Scrumplex](https://github.com/Scrumplex) in [#684](https://github.com/PrismLauncher/PrismLauncher/pull/684)

### Changed

- Detect jars from within build folder for development builds by [@Ryex](https://github.com/Ryex) in [#781](https://github.com/PrismLauncher/PrismLauncher/pull/781)
- Improve the reliability of the Task's stack overflow test by [@flowln](https://github.com/flowln) in [#804](https://github.com/PrismLauncher/PrismLauncher/pull/804)

> - Prevent crash when parsing some Forge mods by [@flowln](https://github.com/flowln) in [#743](https://github.com/PrismLauncher/PrismLauncher/pull/743)

Reviewer: ```suggestion
- Correctly parse metadata from some misbehaving Forge mods by [@flowln](https://github.com/flowln) in [#743](https://github.com/PrismLauncher/PrismLauncher/pull/743)
```
